source/brailleDisplayDrivers/handyTech.py

```diff
@@ -634,6 +692,8 @@ def _set_dotFirmness(self, value):
 		self._dotFirmness = value
 
 	def sendPacket(self, packetType, data=""):
+		if self._dev is None:
+			return

```

Comment:
I'd rather have this return silently if the sleep counter is > 1. If _dev is None and the sleep counter is 0, there's definitely something wrong here and we don't want to miss that.

Also, could you add logging at level debug, telling the user that there is a packet discarded because of the driver being in sleep mode?